postgres-appliance/scripts/configure_spilo.py

```diff
@@ -376,7 +376,7 @@ def get_instance_metadata(provider):
     headers = {}
     if provider == PROVIDER_GOOGLE:
         headers['Metadata-Flavor'] = 'Google'
-        url = 'http://metadata.google.internal/computeMetadata/v1/instance'
+        url = 'http://169.254.169.254/computeMetadata/v1/instance' # metadata.google.internal
```

Comment:
Flake8 complains: postgres-appliance/scripts/configure_spilo.py:379:67: E261 at least two spaces before inline comment
```suggestion
        url = 'http://169.254.169.254/computeMetadata/v1/instance'  # metadata.google.internal
```